### PR TITLE
Add table of contributors to credits section

### PR DESCRIPTION
**Pull request by heuermh**
## PR checklist

- [x] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated
- [ ] If necessary, also make a PR on the [nf-core/pangenome branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/pangenome)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,17 @@ The nf-core/pangenome pipeline comes with documentation about the pipeline which
 
 ## Credits
 
-nf-core/pangenome was originally written by Simon Heumos, Michael L Heuer.
+nf-core/pangenome was originally adapted from the pangenome graph builder [`pggb`](https://github.com/pangenome/pggb) pipeline by Simon Heumos, Michael Heuer.
+
+Many thanks to all who have helped out and contributed along the way, including (but not limited to)\*:
```
Comment (subwaystation) [inline README.md:55]: I am not sure when to bring in the people who currently work on the related MultiQC report module https://github.com/Imipenem/MultiQC/tree/module/odgi_stats/multiqc/modules/odgi @Zethson @Imipenem.

Also, if we take the docker image or Dockerfile from https://github.com/pangenome/pggb, we might want to add @ekg @AndreaGuarracino

Comment (heuermh) [reply README.md:55]: 👍 Please @ekg @AndreaGuarracino  @Zethson @Imipenem add additional lines to this pull request as additional commits, or make follow up pull requests

**Comment by Zethson** [comment]
Done.

Please have a quick look @subwaystation @heuermh

File: README.md (hunk 1 above)
Comment (subwaystation) [reply README.md:55]: Thanks @Zethson for adding everyone.

**Review by subwaystation — APPROVED** [review]
Looks good from my side. @heuermh if you are happy with this, please push the button :+1:

**Comment by heuermh** [comment]
Looks good to me, thanks!  The merge button(s) are disabled for me, not sure if that is because of CI checks or because I don't have permissions to merge.

**Pull request merged**